__tests__/index.test.ts

```diff
@@ -108,6 +109,42 @@ describe('cli', () => {
     versionMock.done();
   });
 
+  describe('logged-in user notifications', () => {
+    let consoleInfoSpy;
+    const getCommandOutput = () => {
+      return [consoleInfoSpy.mock.calls.join('\n\n')].filter(Boolean).join('\n\n');
+    };
+
+    beforeEach(() => {
+      conf.set('email', 'owlbert@readme.io');
+      conf.set('project', 'owlbert');
+      conf.set('apiKey', '123456');
+      consoleInfoSpy = jest.spyOn(console, 'info').mockImplementation();
+    });
+
+    afterEach(() => {
+      consoleInfoSpy.mockRestore();
+      conf.clear();
+    });
+
+    it('should inform a logged in user which project is being updated', async () => {
+      const openapi = new OpenAPICommand();
+      const key = '123456';
+
+      await expect(openapi.run({ key, create: true, update: true })).rejects.toStrictEqual(
+        new Error('The `--create` and `--update` options cannot be used simultaneously. Please use one or the other!')
+      );
+
+      expect(getCommandOutput()).toMatch('is currently logged in, using the stored API key for this project:');
+    });
+
+    it('should not inform a logged in user when they pass their own key', async () => {
+      await expect(cli(['openapi --create --update', '--key=asdf'])).rejects.toThrow('Command not found.');
```

Comment:
That error message doesn't look right 🤔 what happens when you separate out each parameter into separate strings?
```suggestion
      await expect(cli(['openapi', '--create', '--update', '--key=asdf'])).rejects.toThrow('Command not found.');
```